packages/datadog-esbuild/index.js

```diff
@@ -93,11 +93,18 @@ module.exports.setup = function (build) {
   })
 }
 
+// Currently esbuild support requires Node.js >=v16.17 or >=v18.7
+// Better yet it would support Node >=v14.17 or >=v16
+// Of course, the most ideal would be to support all versions of Node that dd-trace supports.
+// Version constraints based on Node's diagnostics_channel support
```

Comment:
Why is that an issue only for this feature but not for other things using DC?

Reply:
I don't know. Haven't had a chance to look into it yet. Fixing the core issue is my next task.